frontend/src/metabase/home/components/Onboarding/OnboardingDismissedToast.tsx

```diff
@@ -0,0 +1,16 @@
+import { t } from "ttag";
+
+import { Stack, Text, Title } from "metabase/ui";
+
+export const OnboardingDismissedToast = () => {
+  return (
+    <Stack spacing="xs">
+      <Title
+        order={4}
+        color="white"
+        fw={700}
+      >{t`Page hidden from the navigation sidebar`}</Title>
```

Comment:
Maybe "Page hidden" is sufficient?

Reply:
> Maybe "Page hidden" is sufficient?

I don't think so because this is very very very specifically tied to the context of removing the page from the main sidebar. I think both the product and the design teams have been over this.

Reply:
https://github.com/metabase/metabase/pull/50536/commits/fdd4fa0da8494e5b347477ac02fb2a8d8b456a47